components/localization/csCZ.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
So these files should be updated manually?

Reply:
No, through the `csde-components-boilerplate-translate` project